requirements.txt

```diff
@@ -4,7 +4,7 @@ Jinja2 ~= 2.10.3
 django-jinja==2.3.1
 django-bootstrap-form==3.2.1
 django-jinja-bootstrap-form ~= 4.2.3
-git+https://github.com/Aalto-LeTech/js-jquery-toggle.git@1.1.0#egg=js-jquery-toggle-django==1.1.0&subdirectory=django
+git+https://github.com/Aalto-LeTech/js-jquery-toggle.git@1.1.2#egg=js-jquery-toggle-django==1.1.0&subdirectory=django
```

Comment:
A mistake in this line. There is different tag compared to python package version requirement. Tag is `1.1.2`, but package requirement is `1.1.0`. I'm not sure how `pip` behaves with this.

Reply:
Is this the correct line?
`git+https://github.com/Aalto-LeTech/js-jquery-toggle.git@1.1.2#egg=js-jquery-toggle-django==1.1.2&subdirectory=django`
I will change this straight away myself.

Reply:
Looks good. [The syntax](https://pip.pypa.io/en/stable/reference/pip_install/#vcs-support) is

```
vcs+protocol://repo_url/#egg=pkg&subdirectory=pkg_dir
```

Where `repo_url` is git reference, i.e., here `https://github.com/Aalto-LeTech/js-jquery-toggle.git@1.1.2` including the tag (as we want specific version from the git and not the "master").

And the `#egg=` contains the "normal" requirements line, i.e.:

> The “project name” component of the URL suffix egg=<project name> is used by pip in its dependency logic to identify the project prior to pip downloading and analyzing the metadata.